designpage.html

```diff
@@ -95,16 +95,16 @@
 			</div>
 			<div class="containthumbs">
 			<div>
-				<img class="img2" onclick="displayImg1Change()"  src="images/image-product-1-thumbnail.jpg"> 
+				<img id="img1" class="img2" onclick='displayImgChange("img1")'  src="images/image-product-1-thumbnail.jpg"> 
 			</div>
 			<div>
-				<img class="img2" onclick="displayImg2Change()" src="images/image-product-2-thumbnail.jpg">
+				<img id="img2" class="img2" onclick='displayImgChange("img2")' src="images/image-product-2-thumbnail.jpg">
 			</div>
 			<div>
-				<img class="img2"  onclick="displayImg3Change()" src="images/image-product-3-thumbnail.jpg">
+				<img id="img3" class="img2"  onclick='displayImgChange("img3")' src="images/image-product-3-thumbnail.jpg">
```

Comment:
CONTINUATION;

but see the src of these ones which i think your function is calling to show on the displayImg id, is their thumbnail jpg file and not the full(original) image files